OpenTrons/opentrons-api
# feat(labware-library, protocol-designer, app): temp deep well adapter…

… support

closes AUTH-77

# Overview

Adds frontend support for the new temp deep well adapter

# Test Plan

NOTE: the svg is still empty and design will be working on creating one!

in PD:
- add a temperature module and go to the deck map, see that the temp deep well adapter can be added to both the deck slot and on top of the temperature module. There should be only 1 labware compatible with it

in App:
- go to the labware tab and see that the adapter is listed with an image

in Labware-library:
- search for the adapter and it should be listed with an image

# Changelog

- add adapter image (i optimized it!) to both LL and app
- add image to the list of labware images in LL and app
- add PD support for the adapter

# Review requests

see test plans

# Risk assessment

low
